src/css/index.css

```diff
@@ -0,0 +1,166 @@
+
+body {
+  margin: 0;
+  padding: 0;
+  font-size: 1rem;
+  line-height: 1.5;
+  font-family: var(--body-family);
+  font-weight: 300;
+}
+
+nav {
+  background: var(--ipfs-gradient-0-background-image);
+}
+nav > div {
+  max-width: var(--max-page-width);
+  margin: 0 auto;
+  padding: 1rem 1rem;
+}
+nav img,
+nav a {
+  vertical-align: middle;
+}
+nav a {
+  color: var(--ipfs-off-white);
+  text-decoration: none;
+  /* font-size: var(--size-h5); */
+  font-weight: var(--weight-semi-bold);
+}
+
+header {
+  background: var(--ipfs-brand-gradient);
+}
+header > div {
+  max-width: var(--max-page-width);
+  margin: 0 auto;
+  padding: 3rem 1rem;
+}
+header h1 {
+  font-size: var(--size-h2);
+  font-weight: var(--weight-semi-bold);
+  margin-top: 0;
+  line-height: 1;
+  color: var(--ipfs-off-white);
+}
+header img.logo {
+  margin-bottom: 2rem;
+}
+header p {
+  color: var(--ipfs-off-white);
+  max-width: var(--max-readable-width);
+}
+
+main {
+  max-width: var(--max-page-width);
+  margin: 4rem auto 0;
+  padding: 0 1rem;
+}
+
+.basic-grid {
+  display: grid;
+  column-gap: 4rem;
+  row-gap: 4rem;
+  grid-template-columns: repeat(1, minmax(0,1fr));
+}
+@media (min-width: 640px) {
+  .basic-grid {
+    grid-template-columns: repeat(2, minmax(0,1fr));
+  }
+}
+
+h2.pitch-title {
+  color: var(--fg);
+  text-align: center;
+  margin-top: 9rem;
+  font-weight: var(--weight-regular);
+  font-size: var(--size-h2);
+  margin-bottom: 1rem;
+  line-height: 1;
+}
+.pitch-subtitle {
+  text-align: center;
+  font-size: var(--size-h5);
+  font-weight: var(--weight-light);
+  margin-bottom: 4rem;
+  line-height: 1.2;
+}
+.pitch {
+  background: var(--stark-bg);
+  color: var(--stark-fg);
+  padding: 0 1rem;
+}
+.pitch h3 {
+  text-align: center;
+  margin-bottom: 1rem;
+}
+.pitch h3::after {
+  width: 3rem;
+  height: 0.5rem;
+  background: var(--ipfs-teal);
+  display: block;
+  content: " ";
+  margin: 0.5rem auto;
+}
+
+main > section {
+  margin-top: 9rem;
+}
+h2 {
+  font-size: var(--size-h2);
+  font-weight: var(--weight-semi-bold);
+  margin-bottom: 1rem;
+}
+main > section section {
+  background: var(--stark-bg);
+  color: var(--stark-fg);
+  padding: 0 1rem;
+}
+h3 {
+  font-size: var(--size-h3);
+  font-weight: var(--weight-semi-bold);
+  margin-bottom: 1rem;
+}
+h3 a {
+  color: inherit;
+}
+
+a {
+  color: var(--highlight-colour);
```

Comment:
:warning:  (so we don't forget) current link color is not accessible, people with vision issues will struggle (see https://github.com/ipfs/public-gateway-checker/issues/342)

probably an easy fix is to pick darker shade from[ the band book](https://www.figma.com/proto/mH0OlgikgKzLmbMNO3noBs/IPFS-Brand-sheet-public?node-id=22-2) (switch to Jade or ligher shade of Navy)

![2023-03-24_01-01](https://user-images.githubusercontent.com/157609/227390359-44c99518-51fe-4853-89ec-35e9e3d4cf2d.png)

Reply:
Adjusted. Requires https://github.com/ipfs/spec-generator/pull/5